### PR TITLE
21479: Update boilerplate repo

### PR DESCRIPTION
**Pull request by devfive**
1. Added examples with `@uxpin*` annotation for properties.
2. Added example with HOC component and `@uxpincomponent` annotation.
3. Added example with `@uxpinnamespace` annotation.
4. Unlocked few eslint flags.
5. Updated all dependencies to actual.
6. Added `yarn.lock` and `package-lock.json`.

### CODE REVIEW

File: src/components/Charts/LineChart/__tests__/__snapshots__/lineChart.tests.js.snap [hunk 1]
```diff
@@ -750,7 +750,7 @@ exports[`renders LineChart with incorrect startData set: enzyme.mount 1`] = `
             }
           >
             <g
-              className="rv-xy-plot__grid-lines"
+              className="rv-xy-plot__grid-lines undefined"
```
Comment (anntr) [inline src/components/Charts/LineChart/__tests__/__snapshots__/lineChart.tests.js.snap:753]: A CO TO TUTAJ TAKIE

File: src/components/Charts/LineMarkChart/__tests__/__snapshots__/lineMarkChart.tests.js.snap [hunk 2]
```diff
@@ -2145,7 +2145,7 @@ exports[`renders LineMarkChart with incorrect startData set: enzyme.mount 1`] =
             }
           >
             <g
-              className="rv-xy-plot__grid-lines"
+              className="rv-xy-plot__grid-lines undefined"
```
Comment (anntr) [inline src/components/Charts/LineMarkChart/__tests__/__snapshots__/lineMarkChart.tests.js.snap:2148]: i tutaj tez takie o

File: src/components/Charts/MarkChart/__tests__/__snapshots__/MarkChart.tests.js.snap [hunk 3]
```diff
@@ -1716,7 +1716,7 @@ exports[`renders MarkChart with incorrect startData set: enzyme.mount 1`] = `
             }
           >
             <g
-              className="rv-xy-plot__grid-lines"
+              className="rv-xy-plot__grid-lines undefined"
```
Comment (anntr) [inline src/components/Charts/MarkChart/__tests__/__snapshots__/MarkChart.tests.js.snap:1719]: i cyk

File: src/components/Charts/LineChart/__tests__/__snapshots__/lineChart.tests.js.snap (hunk 1 above)
Comment (devfive) [reply src/components/Charts/LineChart/__tests__/__snapshots__/lineChart.tests.js.snap:753]: a to przez `react-vis` - `className` jest opcjonalny, ale mimo wszystko jest przekazywany: https://github.com/uber/react-vis/blob/31e9ec1e20558b8c786a8ea3ab8910ec9fbbc1a3/src/plot/grid-lines.js#L140

**Review by anntr — APPROVED** [review]

**Pull request merged**